test/e2e/fixtures/build-default/src/syntax/async.ts

```diff
@@ -0,0 +1,6 @@
+// regression test for async/await
+// code inspired by https://github.com/formium/tsdx/issues/869
+(async () => {
+  await Promise.resolve();
+  console.log('a side effect to make sure this is output');
```

Comment:
ack, since `build-default` is imported/`require`d during the tests to check for correctness, this log actually runs then, producing this output during tests:

```bash
 PASS  test/e2e/tsdx-build-default.test.ts (43.656s)
  ● Console

    console.log stage-build-default/dist/build-default.cjs.development.js:5
      a side effect to make sure this is output
```

Will need to think if I could use a different side-effect that won't print nor potentially break/pollute anything

Reply:
Changed in #903